documentation/dev_running_from_source.markdown

```diff
@@ -159,6 +159,25 @@ run the following:
 
     $ lein run -c /path/to/puppetserver.conf
 
+Running the Agent
+-----
+
+Use a command like the one below to run an agent against your running puppetserver:
+
+        puppet agent --confdir ~/.puppetlabs/etc/puppet \
```

Comment:
Works for me to include the `--confdir` flag.  Still think we might be able to avoid using the `--server` flag in that case, but I'm good either way.